README.md

```diff
@@ -29,3 +29,34 @@ Storage layer with super easy to use in-memory data views and Sherlock-based per
 
 #### [Type Evolution](https://github.com/C5T/Current/blob/master/TypeSystem/Evolution.md)
 Compact and autogenerated C++ framework to evolve objects from one type hierarchy into another type hierarchy while remaining fully within the strong typing paradigm.
+
+## Contribution
+
+Welcome, contributors! Please [start here by signing the CLA](https://github.com/C5T/Current/blob/master/contributors/README.md).
+
+## Quick Start
+
+Install the development dependencies (see below).
+
+Clean, build and run all tests, verify code:
+```
+make clean
+make test
+make check
+```
+
+Build and run tests for each module separately:
+```
+make individual_tests
+```
+
+Build and run tests for one of the modules (e.g. `Blocks/HTTP`):
+```
+( cd Blocks/HTTP && make -j test )
+```
+
+## Development Dependencies
+
+- `nasm` for `FnCAS`. **macOS**: `brew install nasm`
+- `geninfo` from `lcov` for coverage report. **macOS**: `brew install lcov`
+- `clang-format` for code formatting. **macOS**: `brew install clang-format`
```

Comment:
This part is a bit tricky as `clang-format` versions change and they affect formatting. I believe we're on 3.6 now, and it's an open question where and how do we move forward (the newer one messes up the lambdas badly). I'd love to have more clarity here too, but so far we don't quite.